web/src/lib/types/types.ts

```diff
@@ -77,11 +77,8 @@ export type FormationFieldErrorState = {
 
 export const profileFieldsFromConfig = getMergedConfig().profileDefaults.fields;
 
-export type IndustrySpecificDataAddOnFields = "interstateLogistics" | "interstateMoving";
-
 export type ProfileContentField = Exclude<
-  (keyof ProfileData | keyof IndustrySpecificData | IndustrySpecificDataAddOnFields) &
-    keyof typeof profileFieldsFromConfig,
+  (keyof ProfileData | keyof IndustrySpecificData) & keyof typeof profileFieldsFromConfig,
```

Comment:
Technically this is cleanup from the logistics/transport refactor, but I was changing similar code so I thought I'd do it as well. Should have been a separate commit, apologies